### PR TITLE
94 hot wallet channel creation

### PR DESCRIPTION
**Pull request by markettes**

### CODE REVIEW

File: src/Pages/ChannelRequests.razor [hunk 1]
```diff
@@ -662,4 +664,106 @@
...
     }
-}
\ No newline at end of file
+
+    private async void CloseConfirmationModal()
+    {
+        await _confirmationModal.CloseModal();
+    }
+
+    private async void SubmitConfirmationModal()
```
[the remark this answers is not in the document]
Comment (markettes) [reply src/Pages/ChannelRequests.razor]: Done

File: src/Pages/ChannelRequests.razor [hunk 2]
```diff
@@ -662,4 +664,106 @@
             ValidationRule.None(e);
         }
     }
-}
\ No newline at end of file
+
+    private async void CloseConfirmationModal()
```
[the remark this answers is not in the document]
Comment (markettes) [reply src/Pages/ChannelRequests.razor]: Done

File: src/Helpers/LightningHelper.cs [hunk 3]
```diff
@@ -74,8 +75,18 @@ public static void RemoveDuplicateUTXOs(this UTXOChanges utxoChanges)
 
                 foreach (var selectedUtxo in selectedUtxOs)
                 {
-                    var utxoDerivationPath = KeyPath.Parse(key.Path).Derive(selectedUtxo.KeyPath);
-                    var derivedPubKey = bitcoinExtPubKey.Derive(selectedUtxo.KeyPath).GetPublicKey();
+                    KeyPath? utxoDerivationPath;
+                    PubKey derivedPubKey;
+                    if (key.InternalWalletId != null)
```
[the remark this answers is not in the document]
Comment (markettes) [reply src/Helpers/LightningHelper.cs:80]: yes!

**Review by Jossec101 — APPROVED** [review]
LGTM

**Pull request merged**